### PR TITLE
ES6 ✨ commercial third-party-tags IMR worldwide legacy

### PR DESCRIPTION
**Pull request by katebee**
## What does this change?

- ES6 conversion for IMR-worldwide-legacy
- Declare `nol_t` used in imr-worldwide-legacy as a Flow global. Our third-party-tag modules may invoke functions or objects coming from external scripts... Joys of livin' on the edge 😎

## What is the value of this and can you measure success?

Penultimate third-party-tag conversion 🎉
Flow understands what is going on, and stops complaining about `nol_t`

what

## Does this affect other platforms - Amp, Apps, etc?

Nope

### CODE REVIEW

**Review by gu-stav — APPROVED** [review]

File: dev/flow-typed-guardian-frontend/global.js [hunk 1]
```diff
@@ -11,4 +11,7 @@ declare var require: {
     void,
 };
 
-declare var NOLCMB: Object // This is needed for the Nielsen NetRatings tag. Also known as IMR worldwide.
+// These are needed for IMR worldwide, part of Commercial third-party-tags
+declare var NOLCMB: Object; // used in imr-worldwide
+
+declare var nol_t: (config: any) => any;  // used in imr-worldwide-legacy
```
Comment (sndrs) [inline dev/flow-typed-guardian-frontend/global.js]: i think you can declare it in the module if it's not really used elsewhere?

Comment (katebee) [inline dev/flow-typed-guardian-frontend/global.js]: 👍 Cool, will move both declarations to their own modules

Comment (katebee) [inline dev/flow-typed-guardian-frontend/global.js]: @sndrs I've moved the declarations now, how is this? 😊

Comment (sndrs) [inline dev/flow-typed-guardian-frontend/global.js]: gorge 👍

**Review by sndrs — APPROVED** [review]

**Pull request merged**